android/app/src/main/java/com/zulipmobile/MainApplication.java

```diff
@@ -40,7 +41,7 @@
 import static com.zulipmobile.notifications.NotificationHelper.clearConversations;
 
 public class MainApplication extends Application implements ReactApplication, INotificationsApplication {
-    private LinkedHashMap<String, Pair<String, Integer>> conversations;
```

Comment:
Should clean up the now-unused `Pair` import.

Reply:
Done